3-platform/shopping-cart/domain/src/main/resources/application.properties

```
@@ -0,0 +1,3 @@
+spring.r2dbc.url=r2dbc:postgresql://localhost:5432/SIXCORE
```

pls use the same name for the database to avoid misunderstanding among the team. 
check how they named. 
```suggestion
spring.r2dbc.url=r2dbc:postgresql://localhost:5432/postgres
```

---

THX, corrected